### DISCUSSION TITLE
Interpret #tag as tag

### DISCUSSION THREAD

**Discussion by Saturo78** [ransome1/sleek#217] [Ideas]
In the todo.txt android app not only +project and @context are interpreted but also #tag as tag.
It would be really helpful if sleek could work in the same way and it could be also filtered for tags.

Today i use +Projects and @persons tho whom the tasks are assigned and i would like to have a third option for locations, departments or other tags.

**Comment by mshuflin** [comment]
This would be great. I am in the same situation, Sleek on Mac, todo.txt app on android and the #tag feature is nice, but does not work well to not have it available in both situations.

**Reply by ransome1** [reply]
How would we visualise tags in comparison to contexts and projects? As tags are not basic todo.txt elements we would need a different style for those.

**Reply by mshuflin** [reply]
I am not a designer, so I don't feel qualified to suggest how it should look, but it would be great if I could search/filter based on the tags. For example, I want to have a task that says, Discuss +strategicplan (project) with #boss (tag) @work (context)

**Reply by tescaso** [reply]
I would treat tags the same way you do with proyects and contexts, as first level "qualifier", with its own filter section. I am trying to follow GTD method using your app and todo.txt format. In order for this I created special projects for "inbox", "next_action", etc. It would be just perfect if I can use tags for those so I can have a clear distinction between contexts, projects and "tags".
I know I can create different lists for each of these (inbox, etc.) [one todo.txt file for each] but it's not really easy to move tasks between different lists.

**Comment by aisbergde** [comment]
Tags as a third category to organize tasks (#, @, +) would be a good idea, if they will be supported by all apps, used for todo.txt
Currently this is not the case. I don't know, why. Because in General it should be possible. So I ask myself, is there a reason, why they are not used in most todo.txt apps. But some other concept extensions, like threshold, are mostly supported?
